src/components/file/file.mdx

```diff
@@ -6,8 +6,27 @@ menu: Components
 
```

Comment:
You don't have to change it this time, but "documentations" doesn't sound good. It's "documentation" in both singular and plural 🙂